tests/lora/conftest.py

```diff
@@ -152,6 +152,12 @@ def sql_lora_files(sql_lora_huggingface_id):
     return snapshot_download(repo_id=sql_lora_huggingface_id)
 
 
+@pytest.fixture(scope="session")
+def llama3_1_8b_chess_lora():
+    return snapshot_download(
+        repo_id="mkopecki/chess-lora-adapter-llama-3.1-8b")
```

Comment:
@jeejeelee do you have any inputs on how to test LoRA reliably here? Because I haven't be able to find a LoRA that changes Llama 3.1 output in a significant way w/ the  LoRA

Reply:
Generally, I train a LoRA to verify, then compare the outputs between `transformers` and `vllm`

Reply:
Do you have any recommendation what dataset I can use to train a LoRA for Llama 3.1 8b? I tried doing it on Alpaca dataset but I didn't find lot of difference in outputs with and without lora for multiple prompts which I tried

Reply:
TBH, I haven't actually trained a LoRA for audio-LLM. We don't need to rush into training the LoRA right now. Let's focus on completing the related code first

Reply:
Here LoRA is only for language model which is only text. So we don't need to consider audio for now